### DISCUSSION TITLE
can amazon track the use of this logger?

### DISCUSSION THREAD

**Discussion by PieceOfRainbow** [shalzuth/LostArkLogger#87] [Q&A]
I intend to use the data only for myself so I just wanted to know if amazon is capable of tracking the use of this program to strike or ban my account.

**Comment by syntaxerror41** [comment]
Of course they can. That's why all devs involved in the project say use at your own risk. That being said, I highly doubt anyone is getting "permabanned" for using a dps meter, when even infinite chaos botters get 3 day bans.

**Reply by supertakeru** [reply]
https://www.reddit.com/r/lostarkgame/comments/vbqt1v/just_letting_you_guys_know_how_smilegate_deals/ Perma bans are being handed out on KR

**Comment by zhatiana** [comment]
Can you make a trial video?
